src/components/marker-chart/Canvas.js

```diff
@@ -343,11 +386,12 @@ class MarkerChartCanvas extends React.PureComponent<Props, State> {
 
     ctx.fillStyle = '#fff';
     ctx.fillRect(
-      TIMELINE_MARGIN_LEFT,
-      rowIndex * rowHeight - viewportTop + 1, // Add plus one for borders.
-      containerWidth - TIMELINE_MARGIN_LEFT,
-      rowHeight - 2 // Subtract 2 for borders.
+      0,
+      rowIndex * rowHeight - viewportTop,
+      containerWidth,
+      rowHeight - 1 // Subtract 1 for borders.
     );
+    this.drawSeparatorsAndLabels(ctx, rowIndex, rowIndex + 1); // To redraw the Sperator Labels after clearing them
```

Comment:
I'm not entirely sure we should draw the separators and labels here or call it explicitely in drawCanvas. Well I don't know I believe both work :-)

nit: in the comment you wrote `Sperator`.

Reply:
So whenever we clear the row, now we do so starting from the left-most column which is beyond the separator and contains Labels or the Text. After every `clearRow()` the text is erased and filled with `#fff` to re-render the text `drawSeparatorsAndLabels()` is called on that row which is cleared.

Reply:
yeah, sorry if I was confusing: I didn't mean that `drawSeparatorsAndLabels` was unncecessary, it certainly is! My comment was more about _where_ to call it: either in this method, or in `drawCanvas`. But I think this is OK to do it here, so no action needed here :-)